eng/SourceBuildPrebuiltBaseline.xml

```diff
@@ -1,5 +1,16 @@
+<!-- See aka.ms/dotnet/prebuilts for guidance on what pre-builts are and how to eliminate them. -->
 <UsageData>
   <IgnorePatterns>
-    <UsagePattern IdentityGlob="*/*" />
+    <UsagePattern IdentityGlob="Microsoft.SourceBuild.Intermediate.*" />
   </IgnorePatterns>
-</UsageData>
+  <Usages>
+    <!-- Allowed and pinned to major version due to https://github.com/dotnet/source-build/issues/3228 -->
+    <Usage Id="Microsoft.NETCore.App.Crossgen2.linux-x64" Version="8.*" />
+    <Usage Id="Microsoft.NETCore.App.Runtime.linux-x64" Version="8.*" />
+    <!-- Should be removed as part of https://github.com/dotnet/runtime/issues/82298 -->
+    <Usage Id="Microsoft.NETFramework.ReferenceAssemblies" Version="1.0.3" IsDirectDependency="true" IsAutoReferenced="true" />
```

Comment:
These should be excluded with a single exclusion.

 <Usage Id="Microsoft.NETFramework.ReferenceAssemblies*" Version="1.0.3" />

Reply:
Thanks, I somehow missed this